### ISSUE TITLE
Not reading containers correctly

### ISSUE THREAD

**Issue by Irineu333** [Irineu333/SpeakTouch#27]
Comparing to Talkback, some behaviors of Speak Touch don't seem correct regarding containers.

**Context**:
Consider container `ViewGroup` whose what matters to us are their children. When a node with children meets the criteria to be focusable by accessibility but doesn't have `text` or `contentDescription`, the `SpeechInterceptor` reads its children. This solves the problem of custom buttons, such as `CardView` with `TextView` inside.

**Problem**:
I couldn't fully understand the Talkback criteria, so I created my own through trial and error, and due to that, some behaviors are different from Talkback. From these different behaviors, I identified two that seem wrong.

1. It's not reading some nodes with `isCheckable` inside containers when they don't have `text` or `contentDescription`. This is wrong and Talkback is able to read them.

2. It's reading focusable nodes inside containers. Talkback doesn't do that, and I believe it's because these nodes can be individually explored.

**Comment by Irineu333** [comment]
I started considering isCheckable in the isReadable check to fix problem 1.

**Comment by Irineu333** [comment]
I started ignoring the focusable nodes when reading their children to fix problem 2.

**Comment by Irineu333** [comment]
@PatrykMis The switches identified wrong is because of problem 1.
